docker/nginx.conf

```diff
@@ -0,0 +1,32 @@
+user  nginx;
+worker_processes  1;
```

Comment:
This should also be configurable at some point. It will be based on number of cores we give the nginx pods (probably more than 1, less than 8).